feature/results/src/main/java/com/android/developers/androidify/customize/CustomizeExportScreen.kt

```diff
@@ -144,200 +149,254 @@ private fun CustomizeExportContents(
-                    onDownloadClicked()
-                },
-                onWearDeviceClick = {
-                    showWatchFaceBottomSheet = true
-                },
-                hasWearDevice = state.connectedWatch != null,
-                modifier = modifier,
+
+        CustomizeExportLayoutType.Compact -> CustomizeExportScreenScaffold(
+            snackbarHostState,
+            topBar = topBar,
+            containerColor = MaterialTheme.colorScheme.surface,
+        ) { paddingValues ->
+            CustomizeExportLayoutCompact(
+                paddingValues = paddingValues,
+                imageResult = imageResult,
+                state = state,
+                toolSelector = toolSelector,
+                toolDetail = toolDetail,
+                actionButtons = actionButtons,
             )
         }
-        state.connectedWatch?.let { device ->
-            if (showWatchFaceBottomSheet) {
-                WatchFaceModalSheet(
-                    sheetState = watchFaceSheetState,
-                    onDismiss = {
-                        onResetWatchFaceSend()
-                        showWatchFaceBottomSheet = false
-                    },
-                    connectedWatch = device,
-                    installationStatus = state.watchFaceInstallationStatus,
-                    onWatchFaceInstallClick = {
-                        onInstallWatchFaceClicked()
-                    },
-                    onLoad = loadWatchFaces,
-                    watchFaceSelectionState = state.watchFaceSelectionState,
-                    onWatchFaceSelect = onWatchFaceSelect,
-                )
-            }
+
+        CustomizeExportLayoutType.Spatial -> CustomizeExportLayoutSpatial(
+            imageResult = imageResult,
+            state = state,
+            toolSelector = toolSelector,
+            toolDetail = toolDetail,
+            actionButtons = actionButtons,
+            snackbarHostState = snackbarHostState,
+            topBar = topBar,
+        )
+    }
+}
+
+@Composable
+private fun CustomizeExportLayoutCompact(
+    paddingValues: PaddingValues,
+    imageResult: @Composable (ExportImageCanvas.() -> Unit),
+    state: CustomizeExportState,
+    toolSelector: @Composable (Modifier, Boolean) -> Unit,
+    toolDetail: @Composable (Modifier, Boolean) -> Unit,
+    actionButtons: @Composable (Modifier) -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(paddingValues),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.SpaceBetween,
+    ) {
+        Box(
+            modifier = Modifier
+                .weight(1f, fill = true),
+            contentAlignment = Alignment.Center,
+        ) {
+            imageResult(
+                state.exportImageCanvas,
+            )
         }
-        LookaheadScope {
-            CompositionLocalProvider(LocalAnimateBoundsScope provides this) {
```

Comment:
I can't see where this logic has been migrated, is the animation when switching aspect ratios still preserved? 
This should work on XR as it can use a scope within the same layout, it doesn't need to go across window bounds, it performs a grow/shrink animation of the aspect ratio and it's contents as it grows.